addons/website_profile/controllers/main.py

```diff
@@ -84,14 +88,16 @@ def get_user_profile_avatar(self, user_id, field='avatar_256', width=0, height=0
             field_name=field, width=int(width), height=int(height), crop=crop
         ).get_response()
 
-    @http.route(['/profile/user/<int:user_id>'], type='http', auth="public", website=True)
+    @http.route(['/profile/user/<int:user_id>'], type='http', auth='user', website=True)
```

Comment:
I think your commit message, combined with this, is not 100% clear that this was already applicable before you specify it explicitly here, IAMNM. (I.e.; you purpose contains things that are already here).